<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QSlider" name="red_slider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_red">
       <property name="text">
        <string>RED</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="spin_red">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_green">
       <property name="text">
        <string>GREEN</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="spin_green">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSlider" name="green_slider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_blue">
       <property name="text">
        <string>BLUE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="spin_blue">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSlider" name="blue_slider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>red_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>red_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>green_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>green_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blue_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>737</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blue_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>735</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
